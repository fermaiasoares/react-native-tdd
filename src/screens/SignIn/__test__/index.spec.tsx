import React from 'react';
import { render } from '@testing-library/react-native';
import { ThemeProvider } from 'styled-components/native';

import theme from '../../../global/theme';

import { SignIn } from '../';

describe('SignIn Screen', () => {
    it('Should be signin screen match in snapshot', () => {
        const signInScreen = render(
            <ThemeProvider theme={theme}>
                <SignIn />,
            </ThemeProvider>
        );

        expect(signInScreen).toMatchSnapshot();
    })
})