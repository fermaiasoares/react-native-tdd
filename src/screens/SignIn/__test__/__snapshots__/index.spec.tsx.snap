// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Screen Should be signin screen match in snapshot 1`] = `
Array [
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "Roboto_300Light",
            "fontSize": 20,
          },
        ]
      }
    >
      Teste
    </Text>
  </View>,
  ",",
]
`;
